x-pack/plugins/triggers_actions_ui/public/application/components/health_check.test.tsx

```diff
@@ -127,84 +127,6 @@ describe('health check', () => {
     );
   });
 
-  test('renders warning if encryption key is ephemeral', async () => {
```

Comment:
i think we probably want to keep this test? i don't think it's related to ephemeral tasks